lighthouse-core/fraggle-rock/gather/navigation-runner.js

```diff
@@ -287,7 +287,7 @@ async function navigation(options) {
     skipAboutBlank: configContext.skipAboutBlank,
   };
 
-  return Runner.run(
+  const gatherResult = await Runner.gatherPhase(
```

Comment:
Another idea I had was to store artifacts, config, cache, etc on an instance of `Runner` and pass that instance around instead of this new `gatherResult` object.

Reply:
Ya I like this better, gonna see what it looks like.